source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -688,5 +688,37 @@ than an element in the range being searched.
 
 The elements e of [start, end) must be partitioned with respect to the comparator used.
 
+.. code:: cpp
+
+  template <typename Policy, typename InputIt, typename OutputIt, typename UnaryOp,
+      typename UnaryPredicate>
+  OutputIt
+  transform_if(Policy&& policy, InputIt start, InputIt end, OutputIt result, UnaryOp op,
+      UnaryPredicate pred);                                                               // (1)
+
+  template <typename Policy, typename InputIt1, typename InputIt2, typename OutputIt, 
+      typename BinaryOp, typename BinaryPredicate>
+  OutputIt
+  transform_if(Policy&& policy, InputIt1 start1, InputIt1 end1, InputIt2 start2, OutputIt result,
+      BinaryOp op, BinaryPredicate pred);                                                 // (2)
+
+:code:`oneapi::dpl::transform_if` applies a given function to the elements of the input sequence(s) that
+satisfy a given predicate, and stores the result to the output. Depending on the arguments, the algorithm
+
+1. evaluates the unary predicate :code:`pred` for each position :code:`i` of the sequence
```

Comment:
Please capitalize evaluates in steps 1 and 2.

Reply:
both are continuing a sentence started prior to the enumerated list, should we still capitalize them?  Or do you think we should reformat the structure to avoid that?

Reply:
This is a weird one where normal grammar rules don't really apply. Per our and [IDZ's style guide](https://wiki.ith.intel.com/pages/viewpage.action?pageId=1613534372#Formatting&Punctuation-NumberedLists) we format the text using sentence case. We also use a sentence or snippet to introduce the list, so "Depending on the arguments, the algorithm:" is the intro and why we want a colon after "algorithm".

Reply:
alright I've applied that pair of changes.